content/documentation/pages/5-feature-guides/2-batch/4-monitoring.md

```diff
@@ -0,0 +1,170 @@
+
+The core of the Micrometer task integration is part of the Spring Cloud Task’s 2.2.0 release-line, which is a prerequisite for the Task-metrics and the Data Flow integration.
+Any Spring Cloud Task 2.2.0 and newer application can be build and configured to emit Task and Batch metrics to the pre-configured monitoring systems. supported by Micromter.
+
+<!--NOTE-->
+
+To enable Task metrics integration with DataFlow you must add the `spring-boot-starter-actuator` and the selected `micrometer-registry-xxx` dependencies to the Task POM.
+
+<!--END_NOTE-->
+
+For example:
+
+```
+<dependency>
+    <groupId>org.springframework.boot</groupId>
+    <artifactId>spring-boot-starter-actuator</artifactId>
+</dependency>
+<dependency>
+    <groupId>io.micrometer</groupId>
+    <artifactId>micrometer-registry-influx</artifactId>
+</dependency>
+```
+
+To help you get started monitoring streams, Data Flow provides [Grafana](https://grafana.com/) Dashboards that you can install and customize for your needs.
+
+The following image shows the general architecture of how applications are monitored:
+
+![Stream Monitoring Architecture](images/task-metrics-architecture.png)
+
+To allow aggregating metrics per application type and per instance id or per task name, the Spring Cloud Task applications are configured to use the following Micrometer tags:
+
+- `task.name`: The name of the Task that contains the applications that send the metrics
+- `task.execution.id`: [The instance id of the executed task](https://docs.spring.io/spring-cloud-task/docs/2.2.0.M1/reference/#features-generated_task_id).
+- `task.external.execution.id`: The [external Task ID](https://docs.spring.io/spring-cloud-task/docs/2.2.0.M1/reference/#features-external_task_id) as present on the target platform (such as Cloud Foundry or Kubernetes) The type (Source, Processor, or Sink) of the application that reports the metrics
+- `task.parent.execution.id`: The [parent task ID](https://docs.spring.io/spring-cloud-task/docs/2.2.0.M1/reference/#features-parent_task_id) used to identify task that executes another task or tasks.
+
+If the Data Flow server is started with the `spring.cloud.dataflow.grafana-info.url` property pointing to your Grafana URL, the Grafana feature is enabled and Data Flow UI provides you with Grafana-buttons that can open a particular dashboard for a given stream, application, or application instance.
```

Comment:
Since we don't yet have the integration in Task pages, can we please remove this.